src/js/InfoBox.jsx

```diff
@@ -8,8 +8,8 @@ class InfoBox extends Component {
     super(props);
     this.state = {
       currentPrice: null,
-      differenceDollars: null,
-      differencePercentage: null,
+      monthChangeD: null,
```

Comment:
I don't understand why this has changed back

Reply:
These should still be `differenceDollars` and `differencePercentage` throughout this file.